src/llmperf/ray_llm_client.py

```diff
@@ -1,17 +1,12 @@
 import abc
 from typing import Any, Dict, Tuple
 
-from transformers import AutoTokenizer
-
 from llmperf.models import RequestConfig
 
 
 class LLMClient:
     """A client for making requests to a LLM API e.g Anyscale Endpoints."""
 
-    def __init__(self, tokenizer_id="hf-internal-testing/llama-tokenizer"):
-        self.tokenizer = AutoTokenizer.from_pretrained(tokenizer_id)
```

Comment:
Why remove this? Some child classes depend on `tokenizer` being set

Also, now how will `**kwargs` be handled?

Reply:
```
def __init__(self, tokenizer_id: Optional[str] = "hf-internal-testing/llama-tokenizer"):
    if tokenizer_id:
        self.tokenizer = AutoTokenizer.from_pretrained(tokenizer_id)
    else:
        self.tokenizer = None
```